hack/bootstrap-ginkgo.sh

```diff
@@ -0,0 +1,11 @@
+DIRS=$(find pkg/*/ -type d)
+for dir in $DIRS; do
+    # Let's make sure we don't break build by bootstrapping tests in empty
+    # directories.
+    GOFILES=$(find $dir -maxdepth 1 -type f -name *.go | wc -l | xargs)
+    if [ "x${GOFILES}" != "x0" ]; then
+        # Since ginkgo bootstrap doesn't take path, we have to hop in the
+        # target package first.
+        (cd $dir && ginkgo bootstrap || :)
```

Comment:
What came to my mind: How about checking here, if a "*_test.go" file exists in the directory, and only then do the bootstrap?

Reply:
Yeah, that could work. Gonna look into it along the rebase ASAP.